identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f5c1a587",
+   "metadata": {},
+   "source": [
+    "# Notebook - Metrics for Evaluating an Identity Verification Solution \n",
+    "\n",
+    "This notebook provides a walkthrough of the steps mentioned in the blog \"Metrics for Evaluating an Identity Verification Solution\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79de9505",
+   "metadata": {},
+   "source": [
+    "## Step 1 - Data Prep \n",
+    "\n",
+    "1) Upload a dataset of images to an S3 bucket in a single folder. This should include all the images to be tested (both Genuine and Imposter)\n",
+    "\n",
+    "2) In the same bucket, upload an image pair mapping file in CSV format. Ensure that the column headers are exactly the same as is described in Section 1 of the blog.\n",
+    "\n",
+    "The resulting folder structure of your S3 bucket should look like the following:\n",
+    "```\n",
+    "S3 bucket\n",
+    "    │ \n",
+    "    │ -image_pair_mapping_file.csv\n",
+    "    │\n",
+    "    └── dataset/\n",
+    "       │    -file011.jpg\n",
+    "       │    -file012.jpg\n",
+    "       │      .\n",
+    "       │      .\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e86bc8d",
+   "metadata": {},
+   "source": [
+    "### Install python dependencies if necessary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2599ab76",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install boto3 pandas seaborn matplotlib sklearn "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8a339c8",
+   "metadata": {},
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac140c42",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "import json\n",
+    "import io\n",
+    "import pandas as pd\n",
+    "from io import BytesIO\n",
```

Comment:
```suggestion
```

This library doesn't appear to be in any of the following code.

Reply:
This is used in line 122